src/tooltip/test/tooltip-template.spec.js

```diff
@@ -75,5 +75,22 @@ describe('tooltip template', function() {
     expect( elmBody.children().length ).toBe( 1 );
   }));
 
+  it('should open when templateUrl is specified as a string literal rather than an expression - issue 3852', inject(function($compile) {
+    elmBody = angular.element(
+      '<div><span tooltip-template="myUrl">Selector Text</span></div>'
```

Comment:
I think if this change is to be made to support an expression, there should be a breaking change where the attribute value is not assumed to be a string - otherwise this creates a confusing api.